NEWS.md

```diff
@@ -2,6 +2,8 @@
 
 ## Next
 
+* `company-select-first` selects the first completion candidate and
```

Comment:
Please move the descriptions of the commands in the docstrings.

The news entry can simply say that these new commands were added (see some older entries for examples).

Reply:
Done